src/test/java/roomescape/dao/ReservationTimeDaoTest.java

```diff
@@ -44,7 +46,7 @@ public ReservationTimeDaoTest(JdbcTemplate jdbcTemplate) {
         ReservationTime reservationTime = new ReservationTime(request.startAt());
         reservationTimeDao.create(reservationTime);
 
-        reservationTimeDao.delete(2);
+        reservationTimeDao.delete(1);
```

Comment:
```suggestion
                var createdReservationTime = reservationTimeDao.create(reservationTime);

        reservationTimeDao.delete(createdReservationTime.getId());
```

id는 항상 직접 작성하는 것이 아닌 만들어진 객체에서 가져오는 습관을 들이면 유지보수에 편리해요!